scripts/jobs/housing/housing_mtfh_case_notes_enriched.py

```diff
@@ -0,0 +1,109 @@
+        return 0
+
+
+def main():
+    # Set up logging
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
+
+    # parameters
+    glue_client = boto3.client('glue')
+    s3_location_mmh = get_glue_env_var("s3_source_mtfh_notes")
+    mmh_case_notes = get_glue_env_var('source_table_mtfh_notes')
+    s3_location_tenure = get_glue_env_var('s3_source_tenure')
+    tenure_reshaped = get_glue_env_var('source_table_tenure')
+    s3_output_location = get_glue_env_var('s3_output_path')
+
+    # load in datasets
+    tenure_path = get_s3_location(tenure_reshaped, s3_location_tenure)
+    mmh_casenotes_path = get_s3_location(mmh_case_notes, s3_location_mmh)
+    tenure_df = pd.read_parquet(tenure_path)
+    mmh_df = pd.read_parquet(mmh_casenotes_path)
+
+    # tidy up columns
+    mmh_df['create_date'] = pd.to_datetime(mmh_df['createdAt'], utc=True).dt.date
+    mmh_df = mmh_df[['targetId', 'create_date', 'description', 'categorisation']]
+    mmh_df = mmh_df.loc[mmh_df['create_date'] >= (get_date_today_formatted() - timedelta(days=730))]
+    mmh_df.description = mmh_df['description'].astype(str)
+    mmh_df.tenancy_id = mmh_df['targetId'].astype(str)
+    mmh_df['all_notes'] = mmh_df.groupby(['targetId'])['description'].transform(lambda x: '\n'.join(x))
+    tenure_df['start_tenure_date'] = pd.to_datetime(tenure_df['startOfTenureDate'])
+    # tenure_df['uprn'] = tenure_df['uprn'].replace('', 999999).astype('int64')
+    tenure_df['tenancy_id'] = tenure_df['tenancy_id'].astype('str')
+    tenure_df['description'] = tenure_df['description'].astype('str')
+
+    # filter and join tables
+    tenure_df = tenure_df[(tenure_df.endoftenuredate.isnull()) & (tenure_df.asset_type == 'Dwelling') & (
+        tenure_df.description.isin(['Secure', 'Introductory', 'Mesne Profit Ac', 'Mense Profit Ac']))]
+    tenure_df = tenure_df[['tenancy_id', 'start_tenure_date', 'uprn', 'full_address']]
+
+    mmh_notes_df = tenure_df.merge(mmh_df, left_on='tenancy_id', right_on='targetId', how='left')
+    logger.info(f'Dataset size: {mmh_notes_df.shape}\nDataset columns: {mmh_notes_df.columns}')
+
+    # get number of case notes by tenancy
+    mmh_notes_df['num_case_notes'] = mmh_notes_df.groupby(['tenancy_id'])['create_date'].transform('nunique')
+
+    # drop duplicate data and tidy up responses
+    mmh_notes_df = mmh_notes_df.drop_duplicates(subset=['tenancy_id'])
+    mmh_notes_df = mmh_notes_df[['tenancy_id', 'start_tenure_date', 'uprn', 'num_case_notes', 'all_notes']].reset_index(
+        drop=True)
+    mmh_notes_df.all_notes = mmh_notes_df['all_notes'].astype('str').str.strip().str.lower().str.replace('\n', ' ')
+
+    # load spacy dictionary, punctuation and stopwords
+    STOP_WORDS |= {"hackney", "tnt", "tenant", "tenancy", "london", "lbh", "borough", "housing", "monday", "2020",
+                   "2021",
+                   "2023", "intro", "introductory", "property", "team", "email", "date", "tnts", "called", "completed",
```

Comment:
"STOP_WORDS" is static, we'd better to move it to just before your first function (line 17), which will become a global variable, you can cite it in any functions without letting it being an input argument.
The reason is simplifying the main.py and allow this variable to be used in other functions as well.